tests/test_api.py

```diff
@@ -2115,15 +2125,18 @@ def test_logged_in_user(self):
                                   password=self.config.human_password,
                                   http_proxy=self.config.http_proxy)
 
+        sg.update_project_last_accessed(self.project)
         initial = sg.find_one('Project', [['id', 'is', self.project['id']]], ['last_accessed_by_current_user'])
 
+        # Make sure time has elapsed so there is a difference between the two time stamps.
+        time.sleep(2)
+
         sg.update_project_last_accessed(self.project)
 
         current = sg.find_one('Project', [['id', 'is', self.project['id']]], ['last_accessed_by_current_user'])
         self.assertNotEqual(initial, current)
         # it's possible initial is None
-        if initial:
```

Comment:
This test failed again, so I fixed it so it runs exactly the same every single time instead of not running properly the first time and then succeeding the second time on a new project with a new user.